tools/cleanup.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
```

Comment:
Please create a directory for this tool under `tools`. Please also add a `README.md` on how to use it.

Reply:
done